include/nbl/builtin/glsl/ext/RadixSort/default_histogram.comp

```diff
@@ -99,24 +103,38 @@ void nbl_glsl_ext_RadixSort_setHistogram(in uint idx, in uint val)
 #define _NBL_GLSL_EXT_RADIXSORT_HISTOGRAM_SET_HISTOGRAM_DEFINED_
 #endif
 
+#ifndef _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_
+#define _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_ (_NBL_GLSL_WORKGROUP_SIZE_ * 2)
```

Comment:
split into two defines, with the first being `_NBL_GLSL_EXT_RADIXSORT_SHARED_SIZE_NEEDED_`